src/Qir/Runtime/prerequisites.ps1

```diff
@@ -3,21 +3,27 @@
 
 if ($Env:ENABLE_QIRRUNTIME -ne "false") {
     if (($IsWindows) -or ((Test-Path Env:AGENT_OS) -and ($Env:AGENT_OS.StartsWith("Win")))) {
```

Comment:
Minor, but on earlier versions of PowerShell, `$IsWindows` isn't defined. Can either use `#Requires -Version 6.0` or check `$PSVersionTable.PSEdition -eq "Desktop"` to guard against that.